core/script_debugger_remote.cpp

```diff
@@ -1102,6 +1109,10 @@ void ScriptDebuggerRemote::profiling_set_frame_times(float p_frame_time, float p
 	physics_frame_time = p_physics_frame_time;
 }
 
+void ScriptDebuggerRemote::set_skip_breakpoints(bool skip_breakpoints) {
```

Comment:
Should be `p_skip_breakpoints` (thus removing the need for `this->` below).

Reply:
fixed